wicket-intro/CafeAddress/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
try to copy log setup from any other module, we use logback and slf4j as a facade